packages/a-msgpack/src/neat/NeatTypes.ts

```diff
@@ -87,8 +87,7 @@ export class Int {
       }
 
       this.value =
-        parseInt(value, 10) > Number.MAX_SAFE_INTEGER ||
-        parseInt(value, 10) < Number.MAX_SAFE_INTEGER * -1
+        parseInt(value, 10) > 2 ** 31 - 1 || parseInt(value, 10) < 2 ** 31 * -1
```

Comment:
just use the hex values instead of doing the math each time.
`parseInt(value, 10) >= 0x100000000 || parseInt(value, 10) < -0x80000000`